preact.config.js

```diff
@@ -114,14 +112,14 @@ export default function (config, env, helpers) {
 				context: __dirname,
 				from: 'src/_headers',
 				to: '_headers',
-				// Copy-Webpack-Plugin otherwise assumes it's a directory, which results in errors
+				toType: 'file'
+			}, {
+				context: __dirname,
+				from: 'src/_redirects',
+				to: '_redirects',
 				toType: 'file'
 			}]
 		);
-
-		netlifyPlugin(config, {
-			redirects: fs.readFileSync('src/_redirects', 'utf-8').trim().split('\n')
-		});
```

Comment:
Accidentally broken with the v4 migration, simply checked out an older version of the repo to see what the plugin added and it was only a single redirect. Easy enough to just copy into source.